pages/dkp/kommander/2.1/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/kafka/index.md

```diff
@@ -14,6 +14,8 @@ After deploying the Kafka operator, create Kafka Clusters by applying a `KafkaCl
 
 ## Example deployment
 
+<p class="message--note"><strong>NOTE: </strong>The Custom Resources need to be applied to each attached cluster in a project.</p>
```

Comment:
https://github.com/mesosphere/dcos-docs-site/pull/4016/files#diff-0dc2b5b15bb1aae5a27cc7f699afcf39280a74ef0cf4dde2b67fec4324ac42f8R13
>After deploying the Kafka operator, create Kafka Clusters by applying a `KafkaCluster` custom resource in a project's namespace. Refer to the [Kafka operator repository](https://github.com/banzaicloud/koperator/tree/master/config/samples) for examples of the custom resources and their configurations.

Maybe in the paragraph above we can also add
```
...by applying a `KafkaCluster` custom resource **on each attached cluster** in the project's namespace.
```

Reply:
```suggestion
<p class="message--note"><strong>NOTE: </strong>The Custom Resources must be applied to each attached cluster in a project.</p>
```

Reply:
good idea, fixed in d58e0411b0e74dbe4797a6be8f8cc146c1c1019e!